.claude/product-marketing-context.md

```
@@ -0,0 +1,195 @@
+# Product Marketing Context
+
+*Last updated: February 17, 2026*
+
+## Product Overview
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)